services/movie.py

```diff
@@ -15,13 +17,17 @@
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__contains=title)
+
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
```

Comment:
The `get()` method will raise a `DoesNotExist` exception if no movie is found with the given `id`. Consider handling this exception to prevent potential errors.